docker-entrypoint.sh

```diff
@@ -116,9 +116,9 @@ rm -f ${ROOTFS} ${DOCKERFS} ${KUBELETFS}
 truncate -s ${DISK_OS} ${ROOTFS}
 mkfs.xfs ${ROOTFS}
```

Comment:
shouldn't we use ext4 everywhere?

Reply:
Maybe? But I'd like to keep the changes here as small as possible.